search/search-dev-tools/src/components/query/index.js

```diff
@@ -159,10 +156,10 @@ const Query = ( { args, request, url, query_args, backtrace = [] } ) => {
 			<div className={ style.query_res }>
 				<div className={ style.query_result }>
 					<div className={ style.query_actions }>
-						<button id="query-response-copy-handle" data-clipboard-target="#query-response-text" dangerouslySetInnerHTML={ { __html: 'COPY' } }></button>
+						<button id="query-response-copy-handle" data-clipboard-target="#query-response-text">COPY</button>
 					</div>
 					<pre className="line-numbers">
-						<code className="language-json" ref={ queryResultRef } id="query-response-text" dangerouslySetInnerHTML={ { __html: state.result } }></code>
+						<code className="language-json" id="query-response-text">{ state.result }</code>
```

Comment:
1. We don't use the `ref` anywhere, let's drop it.
2. `state.result` is a plain text (the result of `JSON.stringify()`), there is no need to use `dangerouslySetInnerHTML`.